pkg/neotest/basic.go

```diff
@@ -81,24 +81,24 @@ func (e *Executor) PrepareInvoke(t *testing.T, signer interface{},
 }
 
 // SignTx signs a transaction using provided signer.
-// signer can be either *wallet.Account or []*wallet.Account.
+// signer can be either Signer or []Signer.
```

Comment:
Without the need for this.

Reply:
Well, I think varargs is a really good way to handle this, fixed.